EnglishFilter/sections/whitelist.txt

```diff
@@ -144,11 +144,11 @@ swatchseries.to##.download_link_sponsored
 @@||ssgdfm.com^$extension
 ! https://github.com/easylist/easylist/issues/1912
 ! https://github.com/AdguardTeam/AdguardFilters/issues/22746
-|https://$image,xmlhttprequest,domain=pornhub.com|pornhubthbh7ap3u.onion|redtube.com|redtube.com.br|tube8.com|tube8.es|tube8.fr|youporn.com|youporngay.com,badfilter
-@@||phncdn.com^$domain=pornhub.com|pornhubthbh7ap3u.onion,image
-@@||pornhub.com/comment^
-@@||pornhub.com/front^
-@@||pornhub.com/user^
+|https://$image,xmlhttprequest,domain=pornhub.com|pornhub.org|pornhub.net|pornhubthbh7ap3u.onion|redtube.com|redtube.com.br|tube8.com|tube8.es|tube8.fr|youporn.com|youporngay.com,badfilter
```

Comment:
If I'm not wrong, in AdGuard, `$badfilter` modifier disable only exactly the same rule, so I think we should revert it to:
```
|https://$image,xmlhttprequest,domain=pornhub.com|redtube.com|redtube.com.br|tube8.com|tube8.es|tube8.fr|youporn.com|youporngay.com,badfilter
```
because in `EasyList` is rule like this:
```
|https://$image,xmlhttprequest,domain=pornhub.com|redtube.com|redtube.com.br|tube8.com|tube8.es|tube8.fr|youporn.com|youporngay.com
```

Reply:
@DandelionSprout Could you please change it?

Reply:
Duly noted. I've attempted to apply your suggestion now.

Reply:
Thank you.